pyproject.toml

```diff
@@ -54,10 +54,12 @@ dev = [
     "pydata-sphinx-theme>=0.12",
     "pyepics>=3.4.2",
     "pyside6==6.7.0",
+    "pytango>=10.0.0rc1",
```

Comment:
Tango should probably have its own extra set like `ca`, `pva` and `sim`

Reply:
Yes, you want it so that `pip install ophyd-async[tango]` installs `pytango` as a dependency, see the `ca` or `pva` section above here, which add dependencies for the case of `pip install ophyd-async[ca]` and `pip install ophyd-async[pva]`

Reply:
Following up on our discussion involving pytango and numpy -- pytango version 10.x will support numpy 2. It should be the next major release.